aggregator/src/datastore.rs

```diff
@@ -1615,6 +1625,36 @@ impl<C: Clock> Transaction<'_, C> {
         )
     }
 
+    /// Check whether the report has ever been aggregated with the given parameter.
+    #[tracing::instrument(skip(self), err)]
+    pub async fn check_report_aggregation_exists(
```

Comment:
This method should be typesafe for `aggregation_param` (e.g. it should have an `A: Aggregator` generic parameter, and the `aggregation_param` parameter should be an `&A::AggregationParam`).